src/main/java/com/crud/tasks/controller/TaskController.java

```diff
@@ -26,23 +26,44 @@
     }
 
     @RequestMapping(method = RequestMethod.GET, value = "getTask")
-    public TaskDto getTask(@RequestParam("id") Long id) {
-        return taskMapper.mapToTaskDto(dbService.findById(id));
+    public TaskDto getTask(@RequestParam Long taskId) throws TaskNotFoundException {
```

Comment:
chyba o tym juz pisalem - sterowanie wyjatkiem tu mamy. Jak nie ma zadania, to trza zwrocic nulla